stencil.config.ts

```diff
@@ -56,7 +56,10 @@ export const create: () => Config = () => ({
   },
   srcDir: "src/components",
   excludeSrc: DEFAULT_EXCLUDE_SRC,
-  srcIndexHtml: "src/index.html"
+  srcIndexHtml: "src/index.html",
+  extras: {
+    appendChildSlotFix: true
```

Comment:
Do you have a link with more info on this? I couldn't find anything in the Stencil repo.

Reply:
Nope, not yet. Just in the support issue. They will probably have docs once released

Reply:
https://github.com/ionic-team/stencil/commit/e8b4c59261b48295bd787ce10490a928c67ec02f

Reply:
/**
   * By default, the slot polyfill does not update `appendChild()` so that it appends
   * new child nodes into the correct child slot like how shadow dom works. This is an opt-in
   * polyfill for those who need it.
   */
  appendChildSlotFix?: boolean;

Reply:
Got it. Thanks!